lib/master_lock.rb

```diff
@@ -56,6 +56,8 @@ class << self
     # @raise [LockNotAcquiredError] if the lock cannot be acquired before the
     #   timeout
     def synchronize(key, options = {})
+      start_time = Time.now # used for benchmarking
```

Comment:
What time is this? Do you want the amount of time the lock was held for or how long it took to acquire it?